packages/jellyfish-api-core/__tests__/category/blockchain.test.ts

```diff
@@ -55,26 +56,89 @@ describe('masternode', () => {
     await container.stop()
   })
 
-  it('should getBlockchainInfo', async () => {
-    await waitForExpect(async () => {
+  describe('getBlockchainInfo', () => {
+    it('should getBlockchainInfo', async () => {
+      await waitForExpect(async () => {
+        const info = await client.blockchain.getBlockchainInfo()
+        await expect(info.blocks).toBeGreaterThan(1)
+      })
+
       const info = await client.blockchain.getBlockchainInfo()
-      await expect(info.blocks).toBeGreaterThan(1)
+
+      expect(info.chain).toBe('regtest')
+      expect(info.blocks).toBeGreaterThan(0)
+      expect(info.headers).toBeGreaterThan(0)
+
+      expect(info.bestblockhash.length).toBe(64)
+      expect(info.difficulty).toBeGreaterThan(0)
+      expect(info.mediantime).toBeGreaterThan(1550000000)
+
+      expect(info.verificationprogress).toBe(1)
+      expect(info.initialblockdownload).toBe(false)
+      expect(info.chainwork.length).toBe(64)
+      expect(info.size_on_disk).toBeGreaterThan(0)
+      expect(info.pruned).toBe(false)
     })
+  })
 
-    const info = await client.blockchain.getBlockchainInfo()
+  describe('getBlock', () => {
+    const height: number = 1
+    let blockHash: string = ''
+    let verbosity: number = 0
 
-    expect(info.chain).toBe('regtest')
-    expect(info.blocks).toBeGreaterThan(0)
-    expect(info.headers).toBeGreaterThan(0)
+    beforeAll(async () => {
```

Comment:
Since the height is always gonna be 1, it's easier to read if it's just like this.
```ts
    beforeAll(async () => {
      blockHash = await client.blockchain.getBlockHash(1)
    })
```